tests/trainer/test_dataspec.py

```diff
@@ -48,6 +48,27 @@ def test_unable_to_infer(self, dataspec: DataSpec):
             dataspec._default_get_num_samples_in_batch(batch)
 
 
+@pytest.mark.parametrize('batch_size', [4])
+@pytest.mark.parametrize('sequence_length', [8])
```

Comment:
Why are these parameterized if we're not sweeping over?

Reply:
i prefer this over making them consts at the top of the function because it makes it easier to extend the test to different sizes. if you're strongly opposed I can switch.